.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1 +1,32 @@
-**Please refer to the contribution guidelines in the README when submitting PRs.**
+# Description
+
+Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+Fixes # (issue)
+
+## Type of change
+
+Please delete options that are not relevant.
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+
+# How Has This Been Tested?
+
+Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration
+
+- [ ] Test A
+- [ ] Test B
+
+# Checklist:
+
+- [ ] My code follows the style guidelines of this project
```

Comment:
```suggestion
- [ ] My code follows the style guidelines of this project (partially enforced by code linter in CI)
```

Reply:
(Maybe not necessary, but currently we don't have the style guidelines documented very well. E.g., the linter does currently not recommend using type hints, etc.)